pkg/capabilities/consensus/ocr3/reporting_plugin.go

```diff
@@ -139,6 +141,7 @@ func (r *reportingPlugin) Observation(ctx context.Context, outctx ocr3types.Outc
 		allExecutionIDs = append(allExecutionIDs, rq.WorkflowExecutionID)
 	}
 	obs.RegisteredWorkflowIds = r.r.getRegisteredWorkflowsIDs()
+	obs.Timestamp = uint32(time.Now().Unix())
```

Comment:
Protos support timestamps as a native type, can we use that in the Observation proto even if we drop down to uint32 for the actual medianization?